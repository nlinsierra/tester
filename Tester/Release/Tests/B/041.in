2H 4D 5C 8D 5D QH 7D 6S QD AC 2D 6C 7S QS TS 6H 3C 8H 4C KH TD KC 5S AD TH 7C 2C 5H 3D 7H 4S KS 3S 2S AS 9D QC 9H 3H KD 6D TC 4H 8S JS
