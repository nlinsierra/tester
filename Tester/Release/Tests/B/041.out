7
8C 9C JC JD JH AH 9S
